Detectors/TRD/base/include/TRDBase/Calibrations.h

```diff
@@ -66,37 +65,57 @@ class Calibrations
   //
   int const getTimeStamp() { return mTimeStamp; }
   void setTimeStamp(long timestamp) { mTimeStamp = timestamp; }
-  void setCCDB(int calibrationobjecttype);
-  void setCCDBForSimulation() { setCCDB(kSimulation); };
-  void setCCDBForReconstruction() { setCCDB(kReconstruction); };
-  void setCCDBForCalibration() { setCCDB(kCalibration); };
-  //// cant send auto to setCCDB, as a temporary measure, until I can figure something else out. There are 3 formats of getting CCDB parameters defined by the enum at the top.
+  void setCCDB(int calibrationobjecttype, long timestamp);
+  void setCCDBForSimulation(long timestamp) { setCCDB(kSimulation, timestamp); };
+  void setCCDBForReconstruction(long timestamp) { setCCDB(kReconstruction, timestamp); };
+  void setCCDBForCalibration(long timestamp) { setCCDB(kCalibration, timestamp); };
   //
-  //  float getGainMeanRMS();
-  //  float getT0MeanRMS();
-  double getVDrift(long timestamp, int roc, int col, int row) const;
-  double getT0(long timestamp, int roc, int col, int row) const;
-  double getExB(long timestamp, int roc, int col, int row) const;
-  double getGainFactor(long timestamp, int roc, int col, int row) const;
+  double getVDrift(int roc, int col, int row) const;
+  double getT0(int roc, int col, int row) const;
+  double getExB(int roc) const;
+  double getGainFactor(int roc, int col, int row) const;
+  double getPadGainFactor(int roc, int col, int row) const;
+
+  //methods extracted from PadStatus
+  bool isPadMasked(int det, int col, int row) const { return mPadStatus->isMasked(det, col, row); }
+  bool isPadBridgedLeft(int det, int col, int row) const { return mPadStatus->isBridgedLeft(det, col, row); }
+  bool isPadBridgedRight(int det, int col, int row) const { return mPadStatus->isBridgedRight(det, col, row); }
+  bool isPadNotConnected(int det, int col, int row) const { return mPadStatus->isNotConnected(det, col, row); };
+
+  //methods extracted from ChamberStatus
+  bool isChamberGood(int det) const { return mChamberStatus->isGood(det); }
+  bool isChamberNoData(int det, int col, int row) const { return mChamberStatus->isNoData(det); };
```

Comment:
@bazinski, to keep methods consistent with their return methods and uses isChamberNoData should only be function of the detector number and nothing else.

Reply:
@jolopezl well spotted